src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/AddMissingInterpolationTokenTests.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.CodeQuality.CSharp.Analyzers.QualityGuidelines.CSharpAddMissingInterpolationToken,
+    Microsoft.CodeQuality.CSharp.Analyzers.QualityGuidelines.CSharpAddMissingInterpolationTokeFixer>;
+
+namespace Microsoft.CodeQuality.Analyzers.UnitTests.QualityGuidelines
+{
+    public class AddMissingInterpolationTokenTests
```

Comment:
Could we add tests for special cases (see [doc](https://docs.microsoft.com/dotnet/csharp/language-reference/tokens/interpolated#special-characters)):
- escaped curly brace
- ternary operator
- interpolated verbatim string

Reply:
@Evangelink Do we still need the suggested tests given that I'm no longer manually parsing the string, but falling back to speculative semantic model?

Reply:
I would still add them to be sure some know edge cases are covered but I understand if you don't. Maybe that's worth to add at least the test for ternary operator as it forces you to include parenthesis.